test/m1cycles.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Modified from
+ * https://github.com/GMUCERG/PQC_NEON/blob/main/neon/kyber/m1cycles.c, Duc Tri Nguyen (CERG GMU)
```

Comment:
Needs a license entry for third-party code [here](https://github.com/PQCMayo/MAYO-C?tab=readme-ov-file#license)